### PR TITLE
Add a label in the popup widget for the image preview.

### PR DESCRIPTION
**Pull request by mitakas**
The name of the image already has a label, but I often click on an image and expect it to toggle the checkbox.

### CODE REVIEW

File: src/archetypes/referencebrowserwidget/browser/popup.pt [hunk 1]
```diff
@@ -192,9 +192,12 @@
         </td>
 
         <td tal:condition="image_portal_types">
-          <img tal:condition="python: item.Type in image_portal_types"
-               tal:attributes="src string:${item/getURL}/$image_method"
-               />
+          <label tal:condition="referenceable"
```
Comment (frisi) [inline src/archetypes/referencebrowserwidget/browser/popup.pt]: having a condition on the label means: do not show an image for referencable types. shouldn't  we use 

```
<label tal:omit-tag="not: referenceable" ...>
```

to make the image show up even if the item can't be referenced?

**Comment by frisi** [comment]
please also provide a changelog entry

**Comment by mitakas** [comment]
I probably should have rebased my branch first. The conflict is only in docs/CHANGES.txt

**Comment by frisi** [comment]
honestly, i can't help you on how to make your branch automatically mergeable again. i tried git rebase yesterday and ended up with all commits twice.
one advice: create another local branch first (so you've got a backup of your work) and try work on this one.
afaik merging with master will add another unneccessary commit. having spent hours for restoring my commits this night i'd rather go this route though. maybe @keul or @gforcada can give you/us some hints on how to safely rebase your branch.

btw: did you already sign a [plone contributor agreement](http://buildoutcoredev.readthedocs.org/en/latest/agreement.html)?

**Comment by mitakas** [comment]
I synced the master branch of my fork, then rebased the image_preview_label branch and force-pushed the changes. It's OK now.

I haven't sign the plone contributor agreement yet, but will do it now and scan+send it.

**Comment by gbastien** [comment]
Hi @mitakas ,

news regarding Plone contributor agreement? ;-)

I also need this and I would like to see it merged before next release...

Thank you !

Gauthier

**Comment by mitakas** [comment]
Hi @gbastien,

I signed it and emailed it to assignments@plone.org. The physical copy should make it to the USA next Monday.

**Comment by frisi** [comment]
@mitakas is now [listed in the plone organization](https://github.com/orgs/plone/people?query=mitakas). this means the contributor agreement has successfully been processed, right @gbastien?
i'll merge this now. who would be responsible for doing a new release?

**Comment by gbastien** [comment]
@frisi hi,

from what I see on pypi, @esteele or @gotcha could make a new release, could you please?

We will also need a new release of p.a.locales, @vincentfretin could you make one when you will have finished current work on this package?

See you!

Gauthier

**Comment by vincentfretin** [comment]
Sure, ping me when you need the new plone.app.locales release.

**Comment by gbastien** [comment]
Hi @vincentfretin 

a new version was just released (2.5.3) thanks to @esteele , please make a new release of p.a.locales when you come back ;-)

See you,

Gauthier

**Comment by vincentfretin** [comment]
plone.app.locales 4.3.5 released

**Pull request merged**